build/utils/manifest.go

```diff
@@ -0,0 +1,79 @@
+package utils
+
+import (
+	// sha1 is not cryptographically secure
+	// but we just want to generate a reproducible fast hash
+	// to compare the contents of the files
+	// nolint:gosec
+	"crypto/sha256"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"io"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+func GenerateManifest() (string, error) {
+	manifest := ""
```

Comment:
Nit. Would suggest using [strings.Builder](https://pkg.go.dev/strings#Builder) to optimize string concatenation. This is build script so not super important.

Reply:
nice one! I didn't know this existed. Thanks